aries-site/src/examples/components/tabs/TabsExample.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import {
+  Box,
+  Tab,
+  Tabs,
+  Table,
+  TableBody,
+  TableRow,
+  TableCell,
+  Text,
+} from 'grommet';
+
+export const TabsExample = () => {
+  const [index, setIndex] = React.useState();
+  const onActive = nextIndex => setIndex(nextIndex);
+
+  return (
+    <Box>
+      <Tabs activeIndex={index} onActive={onActive} justify="start">
+        <Tab title="General" a11yTitle="Hi there">
```

Comment:
Curious about what the "Hi there" is supposed to communicate / how it is aiming to enhance this example.

Reply:
Whooops, let me remove that. Was a lingering test to make sure aria-labels worked.